.gitignore

```diff
@@ -1,6 +1,9 @@
 # Prerequisites
 *.d
 
+# Shell scripts
+.sh
```

Comment:
Not a part of the onboarding but this helps with the review. It should be `*.sh` and also add `_deps*` and `Testing/`.